pex/pex.py

```diff
@@ -671,6 +671,19 @@ def execute_interpreter(self):
                 sys.argv = args
                 return self.execute_content(arg, content)
         else:
+            if self._vars.PEX_INTERPRETER_HISTORY:
```

Comment:
Unfortunately this pex.py logic also exists independently here for --venv PEXes: https://github.com/pantsbuild/pex/blob/f326835f068088fd9f75929cadf02006f10dfa32/pex/venv/pex.py#L331

That should probably get this same treatment. That main though has no access to the `pex` package.

Reply:
Done! Good tip.